src/graphics/gui.c

```diff
@@ -0,0 +1,23 @@
+#include <stdlib.h>
+
+#include "../include/raylib.h"
+#include "../include/raymath.h"
+
+#include "gui.h"
+
+int screenWidth = 800;
+int screenHeight = 480;
+char title[] = "Raylib Template";
```

Comment:
Change the title string to something more fitting, such as "RTS Minesweeper"

Reply:
This has been done in another PR (https://github.com/SuchithSridhar/RTS-Minesweeper/pull/2) so I've not made it here.